apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -358,7 +359,7 @@ const BookingPage = ({
         hasHashedBookingLink,
         hashedLink,
         smsReminderNumber:
-          selectedLocationType === LocationType.Phone ? booking.phone : booking.smsReminderNumber,
+          selectedLocationType === LocationType.Phone ? booking.phone : booking.smsReminderNumber || "",
```

Comment:
NIT:
<img width="382" alt="image" src="https://user-images.githubusercontent.com/4461358/196273318-4ced1fcc-423d-42e0-97c8-ebb66392c748.png">
Empty strings are saved on DB, this probably be omitted or saved as null like the others.